### PR TITLE
completely blocks IS abusers from entering the game

### PR DESCRIPTION
**Pull request by Amelia0010**
### Intent of your Pull Request

I promised this somewhere at some point

#### Changelog

:cl:  
fix: groudon is now banned
/:cl:

### CODE REVIEW

**Comment by HomocidlePancakes** [comment]
While you are at it ban ktl for being a nerd

File: code/modules/client/client_procs.dm [hunk 1]
```diff
@@ -171,6 +171,12 @@ GLOBAL_LIST_EMPTY(external_rsc_urls)
 
 	GLOB.clients += src
 	GLOB.directory[ckey] = src
+	
+	//Yogs start - fucking internet sounds prevention brigade
+	var/list/ISabusers = list("groudonmaster")
+	if(ckey in ISabusers)
```
Comment (ThatLing) [inline code/modules/client/client_procs.dm:177]: move this to the internet sounds proc

**Review by ThatLing — DISMISSED** [review]

File: code/modules/client/client_procs.dm (hunk 1 above)
Comment (Amelia0010) [reply code/modules/client/client_procs.dm:177]: 🤔

Comment (Amelia0010) [reply code/modules/client/client_procs.dm:177]: no it can potentially be used for events and such, so I'd rather we just have this

**Comment by Amelia0010** [comment]
what no I didn't why the fuck what

**Comment by Kmc2000** [comment]
I actually quite like how you did this.

It gets a merge from me 👍

**Pull request closed without merging**